crates/sparrow-instructions/src/evaluators/aggregation/token/map_accum_token.rs

```diff
@@ -0,0 +1,61 @@
+use arrow::array::{as_map_array, new_empty_array, new_null_array, Array, ArrayRef, MapArray};
+
+use crate::{ComputeStore, StateToken, StoreKey};
+/// Token used for string accumulators.
+///
+/// String accumulators are stored as `[passId, instId, entity_index] ->
+/// Option<String>`. Values are updated entity-by-entity.
+// #[derive(serde::Serialize, serde::Deserialize)]
+pub struct MapAccumToken {
+    /// Stores the state for in-memory usage.
+    // #[serde(with = "sparrow_arrow::serde::array_ref")]
+    accum: MapArray,
```

Comment:
So the `serde` needs to change to be able to support `MapArray`? Wasn't sure of this and haven't looked into it yet.